modules/serverless-logic-overview-error-handling.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+// * about/serverless-logic-overview.adoc
+
+
+:_content-type: CONCEPT
+[id="serverless-logic-overview-error-handling_{context}"]
+= Error handling
+
+Openshift Serverless Logic allows you to define `explicit` error handling, meaning you can define what should happen in case of errors inside your workflow model rather than some generic error handling entity. It enables you to handle the errors that might happen during the interactions between the workflow and external systems. When an error occurs, it changes the regular workflow sequence. In such cases, a workflow state transitions to an alternative state that can potentially handle the error, instead of transitioning to the predefined state.
```

Comment:
```suggestion
OpenShift Serverless Logic allows you to define `explicit` error handling. You can define inside of your workflow model what should happen if errors occur, rather than relying on a generic error handling entity. Explicit error handling enables you to handle the errors that might happen during the interactions between the workflow and external systems. When an error occurs, it changes the regular workflow sequence. In these cases, a workflow state transitions to an alternative state that can potentially handle the error, instead of transitioning to the predefined state.
```

Reply:
Your suggestion is much better and I have incorporated it. :)